purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/conversions/StoreProductConversions.kt

```diff
@@ -142,30 +147,80 @@ private fun ProductVo.createFreeTrialPricingPhase(): PricingPhase? {
     } // This returns null if the string couldn't be parsed to an Int
 }
 
-private fun ProductVo.createPrice(): Price =
-    Price(
+private fun ProductVo.createTieredSubscriptionPricingPhase(): PricingPhase? {
+
+    if (!this.hasTieredSubscription()) { return null }
+    val tieredPrice = this.tieredPrice.toDoubleOrNull() ?: return null
+
+    // tieredSubscriptionCount: If a tiered subscription is available, the number of lower-tier subscription periods
+    val tieredSubscriptionCount = this.tieredSubscriptionCount.toIntOrNull() ?: return null
+    val billingPeriod = createPeriodFromGalaxyData(
+        durationMultiplier = this.tieredSubscriptionDurationMultiplier,
+        durationUnit = this.tieredSubscriptionDurationUnit
+    ) ?: return null
+
+    return PricingPhase(
+        billingPeriod = billingPeriod,
+        recurrenceMode = RecurrenceMode.FINITE_RECURRING,
+        billingCycleCount = tieredSubscriptionCount,
+        price = createPriceFromGalaxyData(
+            currencyUnit = this.currencyUnit,
+            currencyCode = this.currencyCode,
+            itemPrice = tieredPrice
+        ),
+    )
+}
+
+private fun ProductVo.createPrice(): Price = createPriceFromGalaxyData(
+    currencyUnit = this.currencyUnit,
+    currencyCode = this.currencyCode,
+    itemPrice = this.itemPrice,
+)
+
+private fun ProductVo.hasTieredSubscription(): Boolean {
+    // ProductVo.tieredSubscriptionYN: For subscriptions only, whether or not the subscription has two-tiered pricing
+    // "Y": The subscription has one or more lower-price subscription periods followed by regular-price periods
+    // "N": The subscription only has regular-price subscription periods
+    return this.tieredSubscriptionYN == "Y"
+}
+
+private fun createPriceFromGalaxyData(
+    currencyUnit: String,
+    currencyCode: String,
+    itemPrice: Double,
+): Price {
+    return Price(
         // Here, we manually build the formatted string instead of using ProductVo.itemPriceString
         // because itemPriceString doesn't include the decimal values if the amount is an integer with no decimal value.
         // This way, we can get strings like "$3.00" instead of "$3"
         formatted = "%s%.2f".format(currencyUnit, itemPrice),
```

Comment:
I missed this in previous PRs.... But I think we should use `itemPriceString` if available? Note that there are currencies where there are no cents or that it's preferred to show the price without cents when it's a whole number. We could try to handle it ourselves, but I would probably default to what the Galaxy store provides normally.

Reply:
I think that makes sense! Updated to use `itemPriceString` in 9992e979ca617f703b1a0302189e7044792ba4c0